cypress/page/shoping-cart.page.ts

```diff
@@ -2,10 +2,10 @@ class ShopingCartPage {
     private checkbtn: string;
   
     constructor() {
-      this.checkbtn = ".cart_navigation span";
+      this.checkbtn = ".cart_navigation";
```

Comment:
review if you can use the title attribute to indicate you are referring to the proceedToCheckout button

Reply:
done all @holgiosalos